pkg/cluster/kubeadm/config.go

```diff
@@ -51,25 +52,35 @@ func (c *ConfigData) Derive() {
 	}
 }
 
-// DefaultConfigTemplate is the default kubeadm config template used by kind
-const DefaultConfigTemplate = `# config generated by kind
+// DefaultConfigTemplateAlphaV1orV2 is the kubadm config template for
+// API versions v1alpha1 and v1alpha2
+const DefaultConfigTemplateAlphaV1orV2 = `# config generated by kind
 apiVersion: kubeadm.k8s.io/v1alpha2
 kind: MasterConfiguration
+kubernetesVersion: {{.KubernetesVersion}}
+# TODO(bentheelder): fix this upstream!
 clusterName: {{.ClusterName}}
 # on docker for mac we have to expose the api server via port forward,
 # so we need to ensure the cert is valid for localhost so we can talk
 # to the cluster after rewriting the kubeconfig to point to localhost
 apiServerCertSANs: [localhost]
+{{if ne .UnifiedControlPlaneImage ""}}
+# optionally specify a unified control plane image
+unifiedControlPlaneImage: {{.UnifiedControlPlaneImage}}:{{.DockerStableTag}}
+{{end}}`
+
+// DefaultConfigTemplateAlphaV3 is the kubadm config template for
+// API version v1alpha3
+const DefaultConfigTemplateAlphaV3 = `# config generated by kind
+apiVersion: kubeadm.k8s.io/v1alpha3
+kind: ClusterConfiguration
 kubernetesVersion: {{.KubernetesVersion}}
 # TODO(bentheelder): fix this upstream!
-# we need nsswitch.conf so we use /etc/hosts
-# https://github.com/kubernetes/kubernetes/issues/69195
-apiServerExtraVolumes:
-- name: nsswitch
```

Comment:
can you re-introduce the nsswitch bit? I'll follow-up with figuiring out how to remove it in the future, but right now it's required to make cluster up work for k8s master branch

Reply:
missed deleting that too, my bad.